### PR TITLE
`docker-ce` deb meta-package

### PR DESCRIPTION
**Pull request by mikedougherty**
**- What I did**
Created a new build artifact `docker-ce-${VERSION_CE}-0~${DISTRO}.deb` which is processed correctly by the `release-deb` task.

**- How I did it**
Add the requisite `debian/` files: control, rules, compat, files.
Add a new file to contain the CE version
Remove restrictions on which deb files are processed for the repo generation.
Duplicate the steps used to build the docker-engine deb and alter them for docker-ce.

**- How to verify it**
```bash
$ docker volume create --name test-build-repos
$ gpg --gen-key # call it testing, no passphrase -- or adjust ENV vars below
# ...snip
$ make KEEPBUNDLE=1 DOCKER_BUILD_PKGS=ubuntu-xenial deb
# ...snip
$ docker run \
    --rm \
    -i \
    --privileged \
    -e GPG_KEYID=testing \
    -e GPG_PASSPHRASE= \
    -e KEEPBUNDLE=1 \
    -e DOCKER_RELEASE_DIR=/repos \
    -v "$HOME/.gnupg:/root/.gnupg" \
    -v test-build-repos:/repos \
    -v "$(pwd)/bundles:/go/src/github.com/docker/docker/bundles" \
    "docker-dev:ce-deb" hack/make.sh release-deb sign-repos
# ...snip
$ docker run --rm -it -v test-build-repos:/repos ubuntu:xenial bash -c '\
    echo "deb [arch=amd64] file:/repos/apt/repo ubuntu-xenial testing" > /etc/apt/sources.list.d/docker.list ; \
    apt-key add /repos/apt/gpg ; \
    apt update ; \
    apt install -y docker-ce ; \
    docker --version'
# ...snip
Docker version 1.13.0-rc4, build e00ea44
```

**- Description for the changelog**

Add `docker-ce` deb meta-package

**- A picture of a cute animal (not mandatory but encouraged)**
baby ocelot

/cc @icecrime @mlaventure @tiborvass @kencochrane

### CODE REVIEW

File: hack/make/.build-deb-ce/control [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Source: docker-ce
+Section: admin
+Priority: optional
+Maintainer: Docker <support@docker.com>
+Standards-Version: 3.9.6
+Homepage: https://dockerproject.org
```
Comment (kencochrane) [inline hack/make/.build-deb-ce/control]: should we change to https://www.docker.com?

**Review by kencochrane — APPROVED** [review]
LGTM

File: hack/make/.build-deb-ce/control (hunk 1 above)
Comment (mikedougherty) [inline hack/make/.build-deb-ce/control]: good question! i have no opinion -- i just copied it from the other control file :)

if we get another vote for this i'm happy to make the adjustment.

**Comment by mikedougherty** [comment]
closing for now per @vieux

**Pull request closed without merging**